src/rust/engine/src/intrinsics.rs

```diff
@@ -851,14 +855,14 @@ fn parse_python_deps(context: Context, args: Vec<Value>) -> BoxFuture<'static, N
       Level::Debug,
       desc = Some(format!(
         "Determine Python dependencies for {:?}",
-        &prepared_inference_request.path
+        &prepared_inference_request.inner.input_file_path
       )),
       |_workunit| async move {
         let result: ParsedPythonDependencies = get_or_create_inferred_dependencies(
           core,
           &store,
           prepared_inference_request,
-          |content, request| python::get_dependencies(content, request.path),
+          |content, request| python::get_dependencies(content, request.inner.input_file_path.into()),
```

Comment:
Per the PR description, this is the additional String -> PathBuf conversion. We can avoid this in a few ways:

1. use and manipulate `String` in the `get_dependencies` functions directly, instead of `PathBuf`, which seems reasonable given the functions do a `&Path` -> `&str` conversion internally anyway (although this would lose some convenience helpers for manipulating the paths)
2. have a protobuf type that maps to `PathBuf` directly

Or... just not worry about it, because this conversion is fast.